quickstarts/install-netobserv.yaml

```
@@ -0,0 +1,62 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: netobserv
+spec:
+  conclusion: Your Network Observability Operator is installed and configured.
+  description: Install Network Observability Operator and deploy a FlowCollector to capture traffic.
+  displayName: Observe the cluster network traffic
+  durationMinutes: 5
+  introduction: '### In this quick start, you'll deploy and configure Network Observability Operator to get an overview of your cluster network traffic including ingress and egress traffic. For more information on Network Observability, see the related [OpenShift documentation.](https://docs.openshift.com/container-platform/latest/observability/network_observability/network-observability-overview.html)'
+  tags:
+    - network
+    - observability
+    - netobserv
+  tasks:
+    - description: |-
+        You can install the Network Observability Operator using the OpenShift Container Platform web console Operator Hub. When you install the Operator, it provides the FlowCollector custom resource definition (CRD). You can set specifications in the web console when you create the FlowCollector.
+        1. In the OpenShift Container Platform web console, click **OperatorHub** from the [Operators]{{highlight qs-nav-operators}} → OperatorHub.
+        2. Choose **Network Observability Operator** from the list of available Operators in the OperatorHub, and click **Install**.
+        3. Select the checkbox **Enable Operator recommended cluster monitoring on this Namespace**.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: |-
+          #### Verify the operator was successfully installed:
+          Navigate to **Operators** → **Installed Operators**.
+          Is the Network Observability Operator listed?
+      summary:
+        failed: Try the steps again.
+        success: Great work! You successfully installed the Network Observability Operator.
+      title: Install the Network Observability Operator
+    - description: |-
+        In order to start capturing traffic, you need to create a **FlowCollector** resource to configure the components of Network Observability Operator.
+
+        1. Navigate to Operators → Installed Operators and select **Network Observability Operator**.
+
+        2. Navigate to the **FlowCollector** tab, and click **Create FlowCollector**. Make the following selections in the form view:
+          
+          2.1 **spec.agent.ebpf.Sampling**: Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", spec.agent.ebpf.
```

I'd change "sampling size" to "sampling value"
Also maybe just add something like: "a value of 50 means that one packet is sampled every fifty packets".

---

Actually, that's the same in docs, so I guess it's fine ? (cc @skrthomas )
We may also add something like: In a production environment, it is recommended to start with higher values, and gradually decrease it while monitoring the resource utilization.
wdyt? cc @stleerh

---

```suggestion
          2.1 **spec.agent.ebpf.Sampling**: Specify a sampling value for flows. Lower sampling values will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", spec.agent.ebpf.
```

Looks better indeed :ok_hand: